ONSdigital/eq-author-app
# EAR-1355-remove-themes-for-social-surveys

### What is the context of this PR?

Expected behaviour

1 - Open a social survey
2 - Navigate to settings
3 - A tab is not displayed to access the themes settings page
4 - Attempting to access /settings/themes by changing the URL redirects to /settings/general

Actual behaviour

1 - Open a social survey
2 - Navigate to settings
3 - A tab is displayed to access the themes settings page and selecting this tab redirects to /settings/themes with broken content

### How to review

Ensure the expected behaviour now happens.

- Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
  - ensure it can be opened in Author;
  - then, ensure it can be viewed in Runner by pressing the **view survey** button
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
